src/pages/home/report/ReportView.js

```diff
@@ -30,6 +30,7 @@ class ReportView extends React.Component {
                     <ReportActionCompose
                         onSubmit={text => addAction(this.props.reportID, text)}
                         reportID={this.props.reportID}
+                        key={this.props.reportID}
```

Comment:
To help in review:

Basically the `TextInputFocusable` is an uncontrolled input, so when we change the context (reportID) this component have their own value and don't receive the new value from the switch between reportID.

And the change doesn't occur because we sent the value in the `defaultValue` property and when we change the context(reportID) this input already have a value, so `defaultValue` isn't used as a initial value to `value` property.

The solution is when change the context, we could re-render(with `key` property) the component(`ReportActionCompose`) and it will solve this problems.